ods_ci/tests/Resources/Page/FeatureStore/FeatureStore.resource

```diff
@@ -6,11 +6,11 @@ Resource         ../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
 
 
 *** Variables ***
-${FEATURE-STORE-RELEASE-TAG}             v0.52.0
+${FEATURE-STORE-RELEASE-TAG}             master
```

Comment:
Why master and not 0.55.0 ?
The master will have very latest code even code  changes after 0.56.0

Reply:
@jyejare yes 0.55.0 doesn't have notebook template [fix](https://docs.google.com/document/d/14uxT45Bm5qW_4myD9v7EyEfc6mdumD3gW-Ltck6dxSI/edit?tab=t.0#heading=h.jt6ibnbj86zu) so updating master branch. As these will pick just tests so we are good to use master no issues